Makefile

```diff
@@ -342,6 +342,8 @@ endif
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=pca10040 -opt=1     examples/blinky1
 	@$(MD5SUM) test.hex
+	$(TINYGO) build             -o test.elf  -target=nintendoswitch     examples/serial
+	@$(MD5SUM) test.elf
```

Comment:
Sorry to be nitpicking here but you need `test.hex` as output to be able to get a proper md5sum. Otherwise most trivial changes (even a new comment somewhere) may cause a different output.

Reply:
Weird, if I try with `hex` it gives me this error:

```
$ tinygo build             -o test.hex  -target=nintendoswitch     examples/serial
error: ROM segments are non-contiguous: /tmp/tinygo527744533/main
```